wifi/sender/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_package(PkgConfig)
+
+pkg_check_modules(GPS REQUIRED libgps)
+pkg_check_modules(NL libnl-tiny)
```

Comment:
Is this actually a dependency? I was unable to find the correct package to install and it seems to compile just fine without it.

Reply:
Yup, this is just optional. libnl-tiny is for OpenWRT, on a desktop distribution you'd use libnl-3. I'll make fix this so libnl-tiny doesn't show up.